subsystem/cluster_test.go

```diff
@@ -821,6 +821,55 @@ var _ = Describe("cluster install - DHCP", func() {
 	})
 })
 
+var _ = Describe("Create cluster with BaseDNS", func() {
```

Comment:
Use more descriptive name: such validation of baseDNS during create cluster

Reply:
👍🏽